components/checkbox/src/index.js

```diff
@@ -11,7 +11,7 @@ import {
 } from '@govuk-react/constants';
 
 const StyledCheckbox = styled('label')({
-  display: 'block',
+  display: 'table',
```

Comment:
I'm not so sure using display: table is a good idea, is there a fix that feels like less of a hack?

Reply:
I only say this as it is rather unconventional and could have unwanted side effects/trip up future development or people implementing it?

Reply:
also the idea is to mirror GDS styles where possible unless there is a good reason not to, preferably related to React/CSSinJS

I'd recommend wherever we are not mirroring the styles there should be a comment inline explaining the choice

Reply:
That's a good shout actually - using flexbox to solve some problems might be a nice improvement and is easy to add a comment to explain why also.

Reply:
Yes, it is a bit unconventional.. just works! :stuck_out_tongue:

If we go with Flexbox then I think all we need to do is add `display: flex;` to whatever contains the checkboxes. Is there such a container/could there be?

Reply:
My note about flexbox was generic and not specific to this (as there are previous instances where flexbox had been used and was not on GDS) - I might be able to have a look on Thursday! If I can before I will mention on here!